libs/core/src/global/styles/tokens/core/_core.scss

```diff
@@ -106,6 +106,10 @@
   --pine-color-purple-900: #372d82;
   --pine-color-purple-950: #221b4b;
   --pine-font-family-greet: "Greet";
+  --pine-font-size-50: 8px;
+  --pine-font-size-62: 10px;
+  --pine-font-size-75: 12px;
+  --pine-font-size-87: 14px;
   --pine-font-size-100: 16px;
```

Comment:
I think we'll have to adjust the scale if the base font size (100) is going to be changing to 14px.

Reply:
No you're correct, I missed that. Working too quickly here 😅

Will get these updated shortly.